apps/api/src/server/app.js

```diff
@@ -22,6 +23,8 @@ app.use('/', homeRoutes);
 
 app.use('/validation', validationRoutes);
 
+app.use('/appeal-review', appealReviewRoutes);
+
```

Comment:
I feel like we can use the validation routes here. So instead of adding a new folder, that we add a new function to the validation controller to return the appeal details that we need to validate